### PR TITLE
ravisenevirathne/c04-iac01

### PR DESCRIPTION
**Pull request by ravisenevirathne**
# Description

Exercise (e.g. C01-E01): c04-iac01

Student Name: Ravi Senevirathne

> Answers for c04-iac01

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/04class/exercises/c04-iac01/ravisenevirathne/iam.tf [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    resources = [
+      "${aws_s3_bucket.S3.arn}/*"
+    ]
+  }
+
+}
+
+
+/*
+resource "aws_iam_policy" "iam_policy" {
```
Comment (kikobr82) [inline classes/04class/exercises/c04-iac01/ravisenevirathne/iam.tf:32]: I'd recommend you to not add commented code to a main branch.
If you can remove those lines would be great.

**Review by kikobr82 — CHANGES_REQUESTED** [review]
Can you adjust that commented code?

File: classes/04class/exercises/c04-iac01/ravisenevirathne/iam.tf (hunk 1 above)
Comment (ravisenevirathne) [reply classes/04class/exercises/c04-iac01/ravisenevirathne/iam.tf]: Unwanted comments have been removed from iam.tf.   Thank you

**Review by kikobr82 — APPROVED** [review]
lgtm

**Pull request merged**